pwa/src/templates/documentation/usageDocumentation/UsageDocumentationTemplate.module.css

```diff
@@ -11,6 +11,16 @@
   color: var(--web-app-color-text-grey) !important;
 }
 
+.codeblock {
+  --utrecht-code-block-background-color: var(--skeleton-color-grey-1);
+  --utrecht-code-block-padding-block-end: var(--skeleton-size-lg);
+  --utrecht-code-block-padding-block-start: var(--skeleton-size-lg);
+  --utrecht-code-block-padding-inline-end: var(--skeleton-size-lg);
+  --utrecht-code-block-padding-inline-start: var(--skeleton-size-lg);
+  border-radius: var(--skeleton-size-2xs);
+  overflow: auto;
```

Comment:
I think explicitly calling _what_ we expect this overflow to do, might be better. Like so: `overflow-x: scroll`.

Also, at the end of the scroll, we didn't account for any padding, see screenshot:

![image](https://user-images.githubusercontent.com/66552932/227473651-49d2f9bd-3588-48b6-9c08-0af4408de443.png)